corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -295,6 +299,17 @@ def _remote_request_query_datums(self):
                     ref=f"'{','.join(refs)}'",
                 )
             )
+        if module_uses_inline_search_with_parent_relationship_parent_select(self.module):
+            parent_module_id = self.module.parent_select.module_id
+            parent_module = self.app.get_module_by_unique_id(parent_module_id)
+            parent_case_type = parent_module.case_type
+            datums.append(
+                QueryData(
+                    key=CASE_SEARCH_XPATH_QUERY_KEY,
+                    ref=f"ancestor-exists(parent, @case_type={parent_case_type})"
```

Comment:
add string quotes:
```suggestion
                    ref=f"ancestor-exists(parent, @case_type='{parent_case_type}')"
```

Reply:
[ea5b9f3](https://github.com/dimagi/commcare-hq/pull/33738/commits/ea5b9f31cbaa103c4bbb5dc74755eb6b45367bbf) and [f8b91a7](https://github.com/dimagi/commcare-hq/pull/33738/commits/f8b91a76d9b24e2f1bf414cba22b329dc9761139)